ovos_PHAL_plugin_homeassistant/__init__.py

```diff
@@ -25,27 +44,19 @@ def __init__(self, bus=None, config=None):
                 config (dict): The plugin configuration
         """
         super().__init__(bus=bus, name="ovos-PHAL-plugin-homeassistant", config=config)
+        LOG.level = "DEBUG"
         self.oauth_client_id = None
         self.munged_id = "ovos-PHAL-plugin-homeassistant_homeassistant-phal-plugin"
         self.temporary_instance = None
         self.connector = None
-        self.registered_devices = []
+        self.registered_devices = []  # Device objects
+        self.registered_device_names = []  # Device friendly/entity names
```

Comment:
Consider converting this to a property that parses `self.registered_devices` on reference to avoid keeping 2 lists synchronized

Reply:
`self.registered_device_names` gets populated when the device list does, so there's really no synchronization going on

Reply:
I'm not sure how new (or removed) devices are handled on the WS API, but that was my main concern

Reply:
This may have to become part of a larger effort. Right now `self.registered_devices` is a list of device classes, which makes it difficult to search easily. Also currently the list of devices is only built on plugin init, so you'd have to restart PHAL to see new or removed devices. New issue?